axelor-project/src/main/resources/views/Project.xml

```diff
@@ -46,6 +46,8 @@
       <button name="showPlanningBtn" icon="calendar-week" title="Show planning"
         onClick="save,action-project-view-show-planning-panel" if="__config__.app.isApp('employee')"
         if-module="axelor-human-resource" showIf="isShowPlanning"/>
+      <button onClick="action-project-view-attach-tasks-to-sprint"
```

Comment:
Add a save before the action in the onClick and put in next to before the sprintAllocationLinePanel directly in sprint panel + add the icon "link"